openmc/mgxs/mgxs.py

```diff
@@ -1484,22 +1487,26 @@ def rxn_rate_tally(self):
             self._rxn_rate_tally.sparse = self.sparse
         return self._rxn_rate_tally
 
+class FissionXSBase(MGXS):
+    """A fission production multi-group cross section base class
+       for NuFission and KappaFission
+    """
 
-class FissionXS(MGXS):
-    """A fission multi-group cross section."""
+    # This is an abstract class which cannot be instantiated
+    __metaclass__ = abc.ABCMeta
```

Comment:
There is no rule that says that a class you inherit from needs to be abstract.  This is an "intermediate base class"... it injects capability at a certain point in the inheritance hierarchy and doesn't need to be abstract.

No reason to add more complication just to make it "abstract".

If you're really worried about someone creating one of these you have two options:

1.  Just put a comment on it telling people not to do that (this is Python after all... a certain amount of trust in your users is always required)

2.  Move to a Factory pattern for these Tally objects... then just don't allow this class to be built by the Factory.

I vote for #1 ;-)